cn/docs/tutorials/kubernetes-basics/index.html

```diff
@@ -77,7 +77,7 @@
           <div class="thumbnail">
             <a href="/docs/tutorials/kubernetes-basics/scale-intro/"><img src="./public/images/module_05.svg?v=1469803628347" alt=""></a>
             <div class="caption">
-              <a href="scale-intro/"><h5>5. 应用可伸缩</h5></a>
+              <a href="scale-intro/"><h5>5. 应用可扩展</h5></a>
```

Comment:
yeap, but here I think we need to differentiate `scale up` and `scale out`, maybe we can use the translation but keep the English word `scale up` in the bracket